### PR TITLE
Review & implement suggestions to Angular sign-in guide

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## Description:
- **What's changed?** 
  - [vuepress-plugin-code-copy](https://github.com/znicholasbrown/vuepress-plugin-code-copy) added for 'copy into clipboard' button
  - Angular + SIW guide updated according to suggestions
- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-329950](https://oktainc.atlassian.net/browse/OKTA-329950)

### Netlify build:
https://603cfbbdbf1ef7e617ca8926--reverent-murdock-829d24.netlify.app/

### CODE REVIEW

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ If you do not already have a  **Developer Edition Account**, you can create one
 
 ## Add an OpenID Connect Client
 
-* Sign in to the Okta Developer Dashboard, and select **Create New App**
+* Sign in to the Okta Developer Dashboard, and select **Create New App**. <br/>
+You can sign in to Okta Developer Dashboard through [https://login.okta.com](https://login.okta.com) and clicking `Admin` button on Okta Dashboard or by signing in directly into Okta Developer Dashboard through URL which looks like [hxxps://dev-1235813-admin.okta.com](hxxps://dev-1235813-admin.okta.com).
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:24]: @albertchen-okta confirm please, but I don't think we want to have any instructions for the Developer Console?

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ If you do not already have a  **Developer Edition Account**, you can create one
 
 ## Add an OpenID Connect Client
 
-* Sign in to the Okta Developer Dashboard, and select **Create New App**
+* Sign in to the Okta Developer Dashboard, and select **Create New App**. <br/>
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:23]:
```suggestion
* Sign in to the Developer Console, and select **Create New App**. <br/>
```

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 3]
```diff
@@ -20,7 +20,8 @@ If you do not already have a  **Developer Edition Account**, you can create one
...
+You can sign in to the Okta Developer Dashboard through [https://login.okta.com](https://login.okta.com) and clicking `Admin` button on the Okta Dashboard.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:24]:
```suggestion
You can sign in to the Developer Console using [https://login.okta.com](https://login.okta.com), and then click **Admin**.
```

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 4]
```diff
@@ -29,6 +30,7 @@ If you do not already have a  **Developer Edition Account**, you can create one
 | Login redirect URIs  | `http://localhost:4200/login/callback`              |
 | Logout redirect URIs | `http://localhost:4200/login`                       |
 | Allowed grant types  | Authorization Code                                  |
+`${clientId}` placeholders further in this tutorial should be replaced by `Client ID` of created application.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:33]:
```suggestion
`${clientId}` placeholders further in this tutorial should be replaced by the `Client ID` of the created application.
```

**Review by susanharper-okta — APPROVED** [review]
couple knits that I'll commit for you!

**Comment by oleksandrpravosudko-okta** [comment]
Thank you @susanharper-okta!

@shawyu-okta @francoislehoux-okta  are we good with incorporating `code-copy` plugin?

**Comment by francoislehoux-okta** [comment]
Nothing against the `code-copy` plugin. I'll +1 @jakubvul comment here, are we sure we want to be referring the `Dev Console` here? It was turned off just last week. Pulling @albertchen-okta in

**Review by jakubvul-okta — CHANGES_REQUESTED** [review]

**Review by francoislehoux-okta — APPROVED** [review]
Looks good!

File: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md [hunk 5]
```diff
@@ -20,7 +20,8 @@ If you do not already have a  **Developer Edition Account**, you can create one
 
 ## Add an OpenID Connect Client
 
-* Sign in to the Okta Developer Dashboard, and select **Create New App**
+* Sign in to the Admin Console, and select **Create New App**. <br/>
+You can sign in to the Admin Console using [https://login.okta.com](https://login.okta.com), and then click **Admin**.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md:24]:
```suggestion
You can sign in to the Admin Console using <https://login.okta.com>, and then click **Admin**.
```

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**